src/elf.rs

```diff
@@ -462,10 +462,8 @@ impl<'a> Elf<'a> {
         let (from_idx, to_idx) = {
             let to_idx = self.strings.get_or_intern(to);
             let from_idx = self.strings.get_or_intern(from);
-            let (to_idx, _, _) = self.symbols.get_pair_index(&to_idx)
-                .expect(&format!("link could not find symbol pair for \"{}\" at to_idx {}", to, to_idx));
-            let (from_idx, _, _) = self.symbols.get_pair_index(&from_idx)
-                .expect(&format!("link could not find symbol pair for \"{}\" at from_idx {}", from, from_idx));
+            let to_idx = self.symbols.entry(to_idx).index();
+            let from_idx = self.symbols.entry(from_idx).index();
```

Comment:
This change does change the behavior slightly. If either `to_idx` or `from_idx` isn't in `self.symbols`, this silently returns a new index value. Is that intended? Otherwise, it looks like the intended replacement for `get_pair_index` is `get_full`.

Reply:
I believe that with #32 that it is guaranteed to be in the symbols, but, I'll fix that so that in case there is a bug again, it will be caught.